### DISCUSSION TITLE
Ssl certificate

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1117] [Q&A] [answered]
How can I make use of ssl certificate in the client side.. nothing like that in documentation

**Comment by miguelgrinberg** [comment, marked as answer]
You can pass the `http_session` argument to the `Client` object with a session object that is initialized according to your needs. See the documentation at https://python-socketio.readthedocs.io/en/latest/client.html#tls-ssl-support

**Reply by Notadumdum** [reply]
Thanks
